I don't know how to fix this. I think I have to run `make gen` but when I do, I get a SIGSEGV:
```
$ make gen
Building maistra operator
/home/jmazzite/source/maistra/istio-operator/build/generate-crds.sh
Generating CRDs
go: downloading sigs.k8s.io/controller-tools v0.4.1
go: downloading github.com/gobuffalo/flect v0.2.1
go: downloading github.com/spf13/cobra v1.1.1
go: downloading k8s.io/apiextensions-apiserver v0.18.3
go: downloading k8s.io/apimachinery v0.18.3
go: downloading github.com/fatih/color v1.10.0
go: downloading golang.org/x/tools v0.0.0-20210106214847-113979e3529a
go: downloading sigs.k8s.io/yaml v1.2.0
go: downloading k8s.io/api v0.18.3
go: downloading gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
go: downloading github.com/mattn/go-colorable v0.1.8
go: downloading github.com/mattn/go-isatty v0.0.12
go: downloading github.com/spf13/pflag v1.0.5
go: downloading gopkg.in/yaml.v2 v2.3.0
go: downloading golang.org/x/sys v0.0.0-20220715151400-c0bba94af5f8
go: downloading k8s.io/utils v0.0.0-20200603063816-c1c6865ac451
go: downloading golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1
go: downloading golang.org/x/mod v0.3.0
go: downloading k8s.io/klog v1.0.0
go: downloading github.com/google/gofuzz v1.1.0
go: downloading sigs.k8s.io/structured-merge-diff/v3 v3.0.0
go: downloading golang.org/x/net v0.0.0-20201021035429-f5854403a974
go: downloading sigs.k8s.io/structured-merge-diff v1.0.1-0.20191108220359-b1b620dd3f06
go: downloading golang.org/x/text v0.3.3
go: downloading github.com/json-iterator/go v1.1.10
go: downloading github.com/modern-go/reflect2 v1.0.1
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x9d8a5e]

goroutine 70 [running]:
go/types.(*Checker).handleBailout(0xc00065e000, 0xc0004a5d78)
	/home/jmazzite/bin/go-install/go/src/go/types/check.go:434 +0x88
panic({0xc09120?, 0x1363010?})
	/home/jmazzite/bin/go-install/go/src/runtime/panic.go:792 +0x132
go/types.(*StdSizes).Sizeof(0x0, {0xe04570, 0x136bd60})
	/home/jmazzite/bin/go-install/go/src/go/types/sizes.go:229 +0x31e
go/types.(*Config).sizeof(...)
	/home/jmazzite/bin/go-install/go/src/go/types/sizes.go:334
go/types.representableConst.func1(...)
	/home/jmazzite/bin/go-install/go/src/go/types/const.go:77
go/types.representableConst({0xe0af90, 0xe21040}, 0xc00065e000, 0x136bd60, 0xc0004a55d8)
	/home/jmazzite/bin/go-install/go/src/go/types/const.go:93 +0x1e9
go/types.(*Checker).representation(0xc00065e000, 0xc0009054c0, 0x136bd60)
	/home/jmazzite/bin/go-install/go/src/go/types/const.go:257 +0x5f
go/types.(*Checker).implicitTypeAndValue(0xc00065e000, 0xc0009054c0, {0xe04598, 0xc000184850})
	/home/jmazzite/bin/go-install/go/src/go/types/expr.go:374 +0x6d9
go/types.(*Checker).assignment(0xc00065e000, 0xc0009054c0, {0xe04598, 0xc000184850}, {0xcec561, 0x14})
	/home/jmazzite/bin/go-install/go/src/go/types/assignments.go:70 +0x465
go/types.(*Checker).initConst(0xc00065e000, 0xc000f30cc0, 0xc0009054c0)
	/home/jmazzite/bin/go-install/go/src/go/types/assignments.go:144 +0x1d9
go/types.(*Checker).constDecl(0xc00065e000, 0xc000f30cc0, {0xe07ab0, 0xc000413f20}, {0xe07ab0, 0xc0010e0260}, 0x0)
	/home/jmazzite/bin/go-install/go/src/go/types/decl.go:475 +0x2d8
go/types.(*Checker).objDecl(0xc00065e000, {0xe11040, 0xc000f30cc0}, 0x0)
	/home/jmazzite/bin/go-install/go/src/go/types/decl.go:184 +0x98c
go/types.(*Checker).packageObjects(0xc00065e000)
	/home/jmazzite/bin/go-install/go/src/go/types/resolver.go:690 +0x471
go/types.(*Checker).checkFiles(0xc00065e000, {0xc000730f90?, 0x9998a5?, 0xc39b60?})
	/home/jmazzite/bin/go-install/go/src/go/types/check.go:489 +0x2a6
go/types.(*Checker).Files(0xc000486da0?, {0xc000730f90?, 0xc000c02780?, 0x5?})
	/home/jmazzite/bin/go-install/go/src/go/types/check.go:452 +0x75
sigs.k8s.io/controller-tools/pkg/loader.(*loader).typeCheck(0xc00028c8a0, 0xc0004eaa00)
	/home/jmazzite/gopath/pkg/mod/sigs.k8s.io/controller-tools@v0.4.1/pkg/loader/loader.go:283 +0x34a
sigs.k8s.io/controller-tools/pkg/loader.(*Package).NeedTypesInfo(0xc0004eaa00)
	/home/jmazzite/gopath/pkg/mod/sigs.k8s.io/controller-tools@v0.4.1/pkg/loader/loader.go:96 +0x39
sigs.k8s.io/controller-tools/pkg/loader.(*TypeChecker).check(0xc0007ed170, 0xc0004eaa00)
	/home/jmazzite/gopath/pkg/mod/sigs.k8s.io/controller-tools@v0.4.1/pkg/loader/refs.go:263 +0x297
sigs.k8s.io/controller-tools/pkg/loader.(*TypeChecker).check.func1(0x58?)
	/home/jmazzite/gopath/pkg/mod/sigs.k8s.io/controller-tools@v0.4.1/pkg/loader/refs.go:257 +0x4d
created by sigs.k8s.io/controller-tools/pkg/loader.(*TypeChecker).check in goroutine 51
	/home/jmazzite/gopath/pkg/mod/sigs.k8s.io/controller-tools@v0.4.1/pkg/loader/refs.go:255 +0x1aa
exit status 2
make: *** [Makefile:170: generate-crds] Error 1
```

--- a/resources/helm/overlays/istio-telemetry/kiali/templates/kiali-cr.yaml
+++ b/resources/helm/overlays/istio-telemetry/kiali/templates/kiali-cr.yaml
@@ -13,6 +13,7 @@ spec:
     strategy: "openshift"
 
   deployment:
+    cluster_wide_access: false
     accessible_namespaces: []
 {{- if and .Values.kiali.hub .Values.kiali.image }}
     image_name: "{{ .Values.kiali.hub }}/{{ .Values.kiali.image }}"

--- a/resources/helm/v2.6/istio-telemetry/kiali/templates/kiali-cr.yaml
+++ b/resources/helm/v2.6/istio-telemetry/kiali/templates/kiali-cr.yaml
@@ -15,6 +15,7 @@ spec:
     strategy: "openshift"
 
   deployment:
+    cluster_wide_access: false
     accessible_namespaces: []
 {{- if and .Values.kiali.hub .Values.kiali.image }}
     image_name: "{{ .Values.kiali.hub }}/{{ .Values.kiali.image }}"

